### DISCUSSION TITLE
Cases customizion

### DISCUSSION THREAD

**Discussion by dgmyr** [Security-Onion-Solutions/securityonion#10093] [General]
Hi, It's possible to create new custom fields for all tickets in case module?

**Comment by InfosecGoon** [comment]
Probably -- the cases are just documents in the so-case index in Elasticsearch, so they can be modified like any other document. What are you trying to add?

**Reply by dgmyr** [reply]
I wanted to add some custom text or numeric fields which would be displayed in SOC and dashboards in Kibana in order to monitor case statistics.
